Src/GoogleApis.Auth/OAuth2/Flows/GoogleAuthorizationCodeFlow.cs

```diff
@@ -72,12 +78,25 @@ public override AuthorizationCodeRequestUrl CreateAuthorizationCodeRequest(strin
             await DeleteTokenAsync(userId, taskCancellationToken);
         }
 
+        public override bool ShouldForceTokenRetrieval() 
+        {
+            if (includeGrantedScopes.HasValue && includeGrantedScopes.Value == true)

```

Comment:
Please change to: 
return includeGrantedScopes.HasValue && includeGrantedScopes.Value == true;

(if you change the includeGrantedScopes to bool (and not bool?) we can just return includeGrantedScopes; :+1: